app/_src/introduction/concepts.md

```diff
@@ -71,3 +71,17 @@ They can enable traffic management, security, observability and traffic reliabil
 
 Policies always have a clear specific area of impact and goal.
 To learn more about [policies checkout the in depth introduction](/docs/{{ page.release }}/policies/introduction).
+
+### Targets (targetRef)
```

Comment:
Are you planing on using `targets` as a concept in docs? Maybe we can just call this section `targetRef` as it is commonly used in docs and can be easily referenced